app/components/certificate/certificate-display.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Flex,
+  Box,
+  HStack,
+  Text,
+  Heading,
+  Tooltip,
+  IconButton,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+  useToast,
+} from '@chakra-ui/react';
+import { DownloadIcon, CopyIcon } from '@chakra-ui/icons';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  cert: Certificate;
+  priv: Boolean;
+}
+
+export default function CertificateDisplay(props: Props) {
```

Comment:
I would suggest to destruct props, so to do something like `const {isPrivate, certificate} = props` looks cleaner like that

Reply:
Ohhh I never knew that was possible